### PR TITLE
Fix clean RepeateHandler AggregatedobjectIds log message.

### PR DESCRIPTION
**Pull request by tobiasake**
### Applicable Issues
When a Subscription is removed, the repeat handler aggregated objects ids should be cleaned.
If a Subscription has never matched any aggregated objects, then no matched aggregated object ids has been stored in database and the clean up of the Subscriptions matched aggregated objects ids produces and error. This error is not a critical error.
The error message has been improved and describe more whats happening.
Log message has been changed to a log warning message instead.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: src/main/java/com/ericsson/ei/services/SubscriptionService.java [hunk 1]
```diff
@@ -121,8 +121,10 @@ public boolean modifySubscription(Subscription subscription, String subscription
             if (result != null) {
                 String subscriptionIdQuery = String.format(SUBSCRIPTION_ID, subscriptionName);
                 if (!cleanSubscriptionRepeatFlagHandlerDb(subscriptionIdQuery)) {
-                    LOG.error("Failed to clean subscription \"" + subscriptionName
-                            + "\" matched AggregatedObjIds from RepeatFlagHandler database");
+                    LOG.warn("Subscription  \"" + subscriptionName
+                            + "\" matched AggregatedObjIds from RepeatFlagHandler database could not be cleaned during the update of the subscription,\n"
+                    		+ "probably due to subscritpiton has never matched any Aggregated Objects and "
```
Comment (e-pettersson-ericsson) [inline src/main/java/com/ericsson/ei/services/SubscriptionService.java]: subscritpiton  -> subscription

File: src/main/java/com/ericsson/ei/services/SubscriptionService.java [hunk 2]
```diff
@@ -121,8 +121,10 @@ public boolean modifySubscription(Subscription subscription, String subscription
...
+                            + "no matched aggredgatedObjectsIds has been stored in database for the specific subscription.");
```
Comment (e-pettersson-ericsson) [inline src/main/java/com/ericsson/ei/services/SubscriptionService.java]: aggredgatedObjectsIds  -> aggregatedObjectIds

File: src/main/java/com/ericsson/ei/services/SubscriptionService.java [hunk 3]
```diff
@@ -142,8 +144,10 @@ public boolean deleteSubscription(String subscriptionName) throws AccessExceptio
         if (deleteResult) {
             String subscriptionIdQuery = String.format(SUBSCRIPTION_ID, subscriptionName);
             if (!cleanSubscriptionRepeatFlagHandlerDb(subscriptionIdQuery)) {
-                LOG.error("Failed to clean subscription \"" + subscriptionName
-                        + "\" matched AggregatedObjIds from RepeatFlagHandler database");
+                LOG.warn("Subscription  \"" + subscriptionName
+                        + "\" matched AggregatedObjIds from RepeatFlagHandler database could not be cleaned during the removal of subscription,\n"
+                		+ "probably due to subscritpiton has never matched any Aggregated Objects and "
+                        + "no matched aggredgatedObjectsIds has been stored in database for the specific subscription.");
```
Comment (e-pettersson-ericsson) [inline src/main/java/com/ericsson/ei/services/SubscriptionService.java]: Same spelling mistakes in this log message.

File: src/main/java/com/ericsson/ei/services/SubscriptionService.java (hunk 1 above)
Comment (tobiasake) [reply src/main/java/com/ericsson/ei/services/SubscriptionService.java]: Done.

File: src/main/java/com/ericsson/ei/services/SubscriptionService.java (hunk 2 above)
Comment (tobiasake) [reply src/main/java/com/ericsson/ei/services/SubscriptionService.java]: Done.

File: src/main/java/com/ericsson/ei/services/SubscriptionService.java (hunk 3 above)
Comment (tobiasake) [reply src/main/java/com/ericsson/ei/services/SubscriptionService.java]: Done.

**Pull request merged**